src/main/resources/application-seed.yml

```diff
@@ -0,0 +1,4 @@
+spring:
```

Comment:
seed file, this is nice 👍

Reply:
moving it from "local" probably causes the 404s [in the separate `pact` CI job](https://app.circleci.com/pipelines/github/ministryofjustice/hmpps-interventions-service/767/workflows/c03e17e4-3268-4fed-adcb-06ae66966e3c/jobs/2841/steps)